lib/registries/addon/components/sharing-icons/template.hbs

```diff
@@ -0,0 +1,7 @@
+<a class="m-r-xs text-smaller" href={{twitterHref}} onclick={{action 'click' 'link' 'Twitter' 'Content - tweet' extraInfo=resultId target=analytics}}>{{fa-icon 'twitter-square' size=2 }}</a>
+{{!-- facebook app ids are required for sharing and will not always exist --}}
+{{#if facebookHref}} 
```

Comment:
copy pasta from other. We don't do facebook on discover pages, but this allows for sharing icons on providers to add fb as well